test/pack/pack.c

```diff
@@ -502,6 +503,8 @@ int main(int argc, char **argv)
             }
         } else if (!strcmp(*argv, "-verbose")) {
             verbose = 1;
+        } else if (!strcmp(*argv, "-use-tiles")) {
+            use_subdevices = 1;
```

Comment:
I am bothered that we are referring to it with both "tiles" and "subdevices". What is the official term?

Reply:
They are same, and both are used.